src/public/app/components/TopMenu.js

```diff
@@ -14,71 +13,69 @@ const mapStateToProps = state => ({
   currentCountry: state.currentCountry,
   currentGenre: state.currentGenre,
   windowWidth: state.windowWidth,
-})
+});
 
 const mapDispatchToProps = dispatch => ({
-  handleSetCountry: (country) => dispatch({ type: 'SET_CURRENT_COUNTRY', country }),
   handleClearCountry: () => dispatch({ type: 'CLEAR_CURRENT_COUNTRY' }),
-  handleSetGenre: (genre) => dispatch({ type: 'SET_CURRENT_GENRE', genre }),
   handleClearGenre: () => dispatch({ type: 'CLEAR_CURRENT_GENRE' }),
   handleHideCountryDropdown: () => dispatch({ type: 'HIDE_COUNTRY_DROPDOWN' }),
+  handleSetCountry: country => dispatch({ type: 'SET_CURRENT_COUNTRY', country }),
+  handleSetGenre: genre => dispatch({ type: 'SET_CURRENT_GENRE', genre }),
   handleShowCountryDropdown: () => dispatch({ type: 'SHOW_COUNTRY_DROPDOWN' }),
-})
+});
 
 const TopMenu = ({
+  // showUserMenu,
+  // toggleSideMenu,
+  // toggleUserMenu,
   auth,
-  showTopMenu,
-  showFavoritesMenu,
-  showUserMenu,
-  showCountryDropdown,
-  toggleTopMenu,
-  toggleFavoritesMenu,
-  toggleSideMenu,
-  toggleUserMenu,
   currentCountry,
   currentGenre,
-  handleSetCountry,
   handleClearCountry,
+  handleClearGenre,
   handleHideCountryDropdown,
-  handleShowCountryDropdown,
+  handleSetCountry,
   handleSetGenre,
-  handleClearGenre,
+  handleShowCountryDropdown,
+  showCountryDropdown,
+  showFavoritesMenu,
+  showTopMenu,
+  toggleFavoritesMenu,
+  toggleTopMenu,
   windowWidth,
   }) => {
-
   const scrollUp = () => {
     const sweetScroll = new SweetScroll();
     sweetScroll.to(0, 0);
   };
 
-  let height = window.innerHeight - 63;
+  const height = window.innerHeight - 63;
```

Comment:
Possible TODO: cache `window.innerHeight/Width` in redux, and update on resize, to avoid the perf hit while re-rendering components.
https://stackoverflow.com/questions/11486933/why-is-the-performance-of-this-window-innerheight-so-bad